documentation/user-guide/customizing-installation.rst

```diff
@@ -216,6 +216,12 @@ For example in the job XML::
     tree under the :guilabel:`Repos` tab on the distro tree page.
     For example disable_debug_repos.
 
+``disable_onerror``
+    By default ``%onerror`` section is added to kickstart for RHEL7 and newer.
+    In case installation failed this section will handle it and report it to
+    Beaker. Beaker aborts recipe. This option disables this functionality.
+    ``%onerror`` section is not added to kickstart at all.
+
```

Comment:
Suggestion, something like,

Don't add the ``%onerror`` section to the kickstart. By default the section is added for RHEL 7 and newer. It handles the installation failure reporting it to Beaker and aborting the recipe. This option disables this functionality.